lib/instance.js

```diff
@@ -754,13 +754,23 @@ module.exports = (function() {
    * @alias updateAttributes
    */
   Instance.prototype.update = function(values, options) {
+    var changeMemory = this.changed() || []
+      , affectedBySetter
+      , valuesShouldUpdate;
```

Comment:
'changeMemory' => 'changedBefore'
'afftedBySetter' => 'sideEffects'
'valuesShouldUpdate' => 'fields'

Would be my suggestion to names